packages/subgraph/src/packages/token/token-voting.ts

```diff
@@ -111,26 +119,30 @@ export function _handleProposalCreated(
 }
 
 export function handleVoteCast(event: VoteCast): void {
-  let pluginId = event.address.toHexString();
-  let voter = event.params.voter.toHexString();
-  let voterId = pluginId.concat('_').concat(voter);
+  let pluginAddress = event.address;
+  let voterAddress = event.params.voter;
+  let voterEntityId = generateMemberEntityId(pluginAddress, voterAddress);
   let pluginProposalId = event.params.proposalId;
-  let proposalId = getProposalId(event.address, pluginProposalId);
-  let voterVoteId = voter.concat('_').concat(proposalId);
+  let proposalEntityId = generateProposalEntityId(
+    pluginAddress,
+    pluginProposalId
+  );
+  let pluginEntityId = generatePluginEntityId(pluginAddress);
+  let voterVoteEntityId = generateVoteEntityId(voterAddress, proposalEntityId);
   let voteOption = VOTER_OPTIONS.get(event.params.voteOption);
 
   if (voteOption === 'None') {
     return;
   }
 
-  let voterProposalVoteEntity = TokenVotingVote.load(voterVoteId);
+  let voterProposalVoteEntity = TokenVotingVote.load(voterVoteEntityId);
   if (voterProposalVoteEntity) {
     voterProposalVoteEntity.voteReplaced = true;
     voterProposalVoteEntity.updatedAt = event.block.timestamp;
   } else {
-    voterProposalVoteEntity = new TokenVotingVote(voterVoteId);
-    voterProposalVoteEntity.voter = voterId;
-    voterProposalVoteEntity.proposal = proposalId;
+    voterProposalVoteEntity = new TokenVotingVote(voterVoteEntityId);
+    voterProposalVoteEntity.voter = voterEntityId;
```

Comment:
the comment on the `TokenVotingVoter`'s ID shows that the id is an address, and not the output of `generateMemberEntityId()` 

can you verify that is everything is correct and update the schema comment

Reply:
true, good catch

Reply:
re reviewed this and the `generateMemebrEntityId` was correct, in the past we were using the generated id, not the address id